### DISCUSSION TITLE
[5.x] "No entries yet" message for Matrix fields should be customizable to reflect “New” Button Label verbiage

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#14797] [Ideas] [answered]
We're using a Matrix field for entries we refer to as "Modules" so we've customized the "New" Button Label option to say "New module." This looks a little disjointed with the new "No entries yet" messaging, so it'd be nice to be able to customize that messaging as well.
![2024-04-11 at 16 26 27](https://github.com/craftcms/cms/assets/42223501/5588b243-cc5e-4df6-b028-29c2b1cc1b8f)

**Comment by brandonkelly** [comment, marked as answer]
I just modified that to say “Nothing yet.” for Craft 5.1, sortof avoiding the discrepancy while also making it sound slightly friendlier. (7cf8f8d42c8fbb58a6fffc150bd7fb6b94e976d3)
